backend/routes/searchExternalUserRouter.js

```diff
@@ -16,15 +16,22 @@ const controller = ({ oauthApi }) => {
     pageSize: size,
   }) => oauthApi.userSearch(context, { nameFilter, group: groupCode, role: roleCode, status }, page, size)
 
+  const assignableGroups = async (context) =>
+    (await oauthApi.assignableGroups(context)).map((g) => ({
```

Comment:
translation moved here to `text` and `value` as caseload version has `agencyId` and `description` instead